CHANGELOG.md

```diff
@@ -4,6 +4,81 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
```

Comment:
I know this is not part of the ticket, but is there any chance we can update the intro sentence to say:

This page lists all the latest features and updates to TimescaleDB. When you use `psql` to update your database, use  `-X` flag and prevent any `.psqlrc` commands from accidentally triggering the load of a previous DB version.

Reply:
@billy-the-fish please make a suggestion in the file